sys/include/net/gnrc/rpl.h

```diff
@@ -82,6 +82,40 @@
  *   CFLAGS += -DCONFIG_GNRC_RPL_WITHOUT_VALIDATION
  *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
+ * - This RPL implementation currently only supports storing mode.
+ *   That means, in order to have downwards routes to all nodes the
+ *   storage space within [`gnrc_ipv6`'s Neighbor Information Base](@ref net_gnrc_ipv6_nib)
```

Comment:
Markdown code markers (`) in a link [don't seem to work](https://3826-7370241-gh.circle-artifacts.com/0/doc/group__net__gnrc__rpl.html) :-( Then let's just use Doxygen markup:

```suggestion
 *   storage space within [@c gnrc_ipv6's Neighbor Information Base](@ref net_gnrc_ipv6_nib)
```